openassessment/xblock/test/data/invalid_update_xblock.json

```diff
@@ -1403,7 +1403,7 @@
                 "due": null
             }
         ],
-        "editor_assessments_order": ["student-training", "peer-assessment", "self-assessment", "example-based-assessment", "staff-assessment"],
+        "editor_assessments_order": ["example-based-assessment", "student-training", "peer-assessment", "self-assessment", "staff-assessment"],
```

Comment:
The changes to validation have made "Verify that the new `editor_assessments_order` is valid" be checked before the non-unique option names this test is supposed to be checking, so I updated the order to a valid one.